src/dev/dev_command.ts

```diff
@@ -55,7 +55,7 @@ export async function dev(
     // Ensure that build dir is empty
     await fs.emptyDir(outDir);
     await build(join(dir, "fresh.gen.ts"), options);
-  } else if (options) {
+  } else if (Object.keys(options).length) {
```

Comment:
I think what this check was supposed to do, was check if a third argument was passed to the function. Passing an empty object is valid and should fall into this branch too.

Reply:
In the v1.4.3, the third argument defaults to `{}`, so the following code is unreachable code.
```ts
await import(entrypoint);
```
`Boolean({})` is `true`.

Reply:
Exactly, that's the issue.

Reply:
I have updated the fix.